examples/sn-react-imagegallery/src/app.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useEffect, useState } from 'react'
+import { ConstantContent } from '@sensenet/client-core'
+import moment from 'moment'
+import { CssBaseline, Slide } from '@material-ui/core'
+import { TransitionProps } from '@material-ui/core/transitions'
+import { makeStyles } from '@material-ui/core/styles'
+import { Image, User } from '@sensenet/default-content-types'
+import snLogo from './assets/sensenet_logo_transparent.png'
+import { useRepository } from './hooks/use-repository'
```

Comment:
Please refactor this to use `useRepository` from @sensenet/hooks-react in all components.